common/app/experiments/Experiments.scala

```diff
@@ -19,7 +19,7 @@ object LiveblogRendering
       description = "Use DCR for liveblogs",
       owners = Seq(Owner.withGithub("shtukas")),
       sellByDate = LocalDate.of(2022, 6, 2),
-      participationGroup = Perc10A,
+      participationGroup = Perc1A,
```

Comment:
Do we still want to do this given what we know about dead vs. live traffic?

Reply:
I'd probably stick to this, super easy to ramp up and if something does go wrong, we're going to make it harder for ourselves.

It's not a confidence thing, just a that-one-thing-we-didn't-think-of thing.

Reply:
We'll put it in the comms we plan to move to 10/50 quite quickly.

Reply:
I was less thinking about confidence - I agree we should be cautious. It was more: will we have sufficient data for the metrics? Wasn't there some comms about an issue around this? Maybe it's a question for D&I?

Reply:
If there was I've missed it. The concern being we won't have enough data for...?

Reply:
There’s a way to override tests for commercial metrics. Commercial are proposing to add the same thing in DCR here: https://github.com/guardian/dotcom-rendering/pull/4288

In frontend, this is done in https://github.com/guardian/frontend/blob/main/static/src/javascripts/projects/common/modules/analytics/shouldCaptureMetrics.ts#L5-L10